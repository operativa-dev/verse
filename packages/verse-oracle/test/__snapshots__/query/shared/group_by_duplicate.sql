-- Executing SQL: Parameters: []
select t1.ArtistId, t1.ArtistId c0
from Album t1
group by t1.ArtistId

