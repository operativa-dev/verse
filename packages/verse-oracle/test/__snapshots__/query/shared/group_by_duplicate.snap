[
  [
    147,
    147,
  ],
  [
    148,
    148,
  ],
  [
    149,
    149,
  ],
  [
    150,
    150,
  ],
  [
    151,
    151,
  ],
  [
    152,
    152,
  ],
  [
    153,
    153,
  ],
  [
    72,
    72,
  ],
  [
    155,
    155,
  ],
  [
    156,
    156,
  ],
  [
    157,
    157,
  ],
  [
    158,
    158,
  ],
  [
    159,
    159,
  ],
  [
    114,
    114,
  ],
  [
    179,
    179,
  ],
  [
    180,
    180,
  ],
  [
    36,
    36,
  ],
  [
    196,
    196,
  ],
  [
    197,
    197,
  ],
  [
    198,
    198,
  ],
  [
    199,
    199,
  ],
  [
    200,
    200,
  ],
  [
    201,
    201,
  ],
  [
    202,
    202,
  ],
  [
    203,
    203,
  ],
  [
    204,
    204,
  ],
  [
    205,
    205,
  ],
  [
    8,
    8,
  ],
  [
    206,
    206,
  ],
  [
    207,
    207,
  ],
  [
    208,
    208,
  ],
  [
    209,
    209,
  ],
  [
    210,
    210,
  ],
  [
    211,
    211,
  ],
  [
    212,
    212,
  ],
  [
    213,
    213,
  ],
  [
    214,
    214,
  ],
  [
    215,
    215,
  ],
  [
    216,
    216,
  ],
  [
    217,
    217,
  ],
  [
    218,
    218,
  ],
  [
    219,
    219,
  ],
  [
    220,
    220,
  ],
  [
    221,
    221,
  ],
  [
    222,
    222,
  ],
  [
    223,
    223,
  ],
  [
    224,
    224,
  ],
  [
    225,
    225,
  ],
  [
    226,
    226,
  ],
  [
    227,
    227,
  ],
  [
    228,
    228,
  ],
  [
    229,
    229,
  ],
  [
    230,
    230,
  ],
  [
    231,
    231,
  ],
  [
    232,
    232,
  ],
  [
    233,
    233,
  ],
  [
    234,
    234,
  ],
  [
    235,
    235,
  ],
  [
    236,
    236,
  ],
  [
    237,
    237,
  ],
  [
    238,
    238,
  ],
  [
    240,
    240,
  ],
  [
    241,
    241,
  ],
  [
    242,
    242,
  ],
  [
    243,
    243,
  ],
  [
    244,
    244,
  ],
  [
    245,
    245,
  ],
  [
    246,
    246,
  ],
  [
    247,
    247,
  ],
  [
    248,
    248,
  ],
  [
    249,
    249,
  ],
  [
    250,
    250,
  ],
  [
    251,
    251,
  ],
  [
    252,
    252,
  ],
  [
    253,
    253,
  ],
  [
    254,
    254,
  ],
  [
    255,
    255,
  ],
  [
    256,
    256,
  ],
  [
    257,
    257,
  ],
  [
    258,
    258,
  ],
  [
    259,
    259,
  ],
  [
    260,
    260,
  ],
  [
    261,
    261,
  ],
  [
    262,
    262,
  ],
  [
    263,
    263,
  ],
  [
    264,
    264,
  ],
  [
    265,
    265,
  ],
  [
    266,
    266,
  ],
  [
    267,
    267,
  ],
  [
    268,
    268,
  ],
  [
    269,
    269,
  ],
  [
    270,
    270,
  ],
  [
    271,
    271,
  ],
  [
    272,
    272,
  ],
  [
    273,
    273,
  ],
  [
    274,
    274,
  ],
  [
    275,
    275,
  ],
  [
    1,
    1,
  ],
  [
    2,
    2,
  ],
  [
    3,
    3,
  ],
  [
    4,
    4,
  ],
  [
    5,
    5,
  ],
  [
    6,
    6,
  ],
  [
    7,
    7,
  ],
  [
    9,
    9,
  ],
  [
    10,
    10,
  ],
  [
    11,
    11,
  ],
  [
    12,
    12,
  ],
  [
    13,
    13,
  ],
  [
    14,
    14,
  ],
  [
    15,
    15,
  ],
  [
    16,
    16,
  ],
  [
    17,
    17,
  ],
  [
    18,
    18,
  ],
  [
    19,
    19,
  ],
  [
    20,
    20,
  ],
  [
    21,
    21,
  ],
  [
    22,
    22,
  ],
  [
    23,
    23,
  ],
  [
    24,
    24,
  ],
  [
    50,
    50,
  ],
  [
    51,
    51,
  ],
  [
    52,
    52,
  ],
  [
    53,
    53,
  ],
  [
    54,
    54,
  ],
  [
    55,
    55,
  ],
  [
    56,
    56,
  ],
  [
    57,
    57,
  ],
  [
    58,
    58,
  ],
  [
    59,
    59,
  ],
  [
    37,
    37,
  ],
  [
    68,
    68,
  ],
  [
    69,
    69,
  ],
  [
    70,
    70,
  ],
  [
    76,
    76,
  ],
  [
    77,
    77,
  ],
  [
    78,
    78,
  ],
  [
    79,
    79,
  ],
  [
    80,
    80,
  ],
  [
    41,
    41,
  ],
  [
    81,
    81,
  ],
  [
    82,
    82,
  ],
  [
    83,
    83,
  ],
  [
    84,
    84,
  ],
  [
    85,
    85,
  ],
  [
    86,
    86,
  ],
  [
    27,
    27,
  ],
  [
    87,
    87,
  ],
  [
    88,
    88,
  ],
  [
    89,
    89,
  ],
  [
    90,
    90,
  ],
  [
    91,
    91,
  ],
  [
    92,
    92,
  ],
  [
    93,
    93,
  ],
  [
    94,
    94,
  ],
  [
    95,
    95,
  ],
  [
    46,
    46,
  ],
  [
    96,
    96,
  ],
  [
    97,
    97,
  ],
  [
    98,
    98,
  ],
  [
    99,
    99,
  ],
  [
    100,
    100,
  ],
  [
    101,
    101,
  ],
  [
    102,
    102,
  ],
  [
    103,
    103,
  ],
  [
    104,
    104,
  ],
  [
    105,
    105,
  ],
  [
    42,
    42,
  ],
  [
    106,
    106,
  ],
  [
    108,
    108,
  ],
  [
    109,
    109,
  ],
  [
    110,
    110,
  ],
  [
    111,
    111,
  ],
  [
    112,
    112,
  ],
  [
    113,
    113,
  ],
  [
    115,
    115,
  ],
  [
    116,
    116,
  ],
  [
    117,
    117,
  ],
  [
    118,
    118,
  ],
  [
    120,
    120,
  ],
  [
    121,
    121,
  ],
  [
    122,
    122,
  ],
  [
    124,
    124,
  ],
  [
    125,
    125,
  ],
  [
    126,
    126,
  ],
  [
    127,
    127,
  ],
  [
    128,
    128,
  ],
  [
    130,
    130,
  ],
  [
    131,
    131,
  ],
  [
    132,
    132,
  ],
  [
    133,
    133,
  ],
  [
    134,
    134,
  ],
  [
    135,
    135,
  ],
  [
    136,
    136,
  ],
  [
    137,
    137,
  ],
  [
    138,
    138,
  ],
  [
    139,
    139,
  ],
  [
    140,
    140,
  ],
  [
    141,
    141,
  ],
  [
    142,
    142,
  ],
  [
    143,
    143,
  ],
  [
    144,
    144,
  ],
  [
    145,
    145,
  ],
  [
    146,
    146,
  ],
]